star-wars-browser/src/components/search_results/SearchResults.jsx

```diff
@@ -38,4 +36,14 @@ const SearchResults = () => {
   );
 };
 
+SearchResults.defaultProps = {
+  pageNumber: () => 1,
```

Comment:
page number powinno być liczbą, a jest funkcją zwracającą liczbę

change page domyślnie zwraca funkcję która wypisuje "working" - to też chyba można usunąć

rozważ, czy one muszą mieć domyślne wartości - czy ten komponent ma sens bez pageNumber i changePage?

jeśli muszą zostać, to lepiej, żeby change page było pustą funkcją, czyli () =>{}